app/(server)/api/events/[slug]/route.ts

```diff
@@ -17,8 +18,19 @@ export async function GET(
     let userID = undefined;
 
     if (requestToken) {
-      const response = await getCurrentAuthenticated(requestToken.value);
-      if (response.ok) userID = response.data.id;
+      const response: AuthType.CurrentAuthResponse =
+        await InternalApiFetcher.get('/api/auth/current', {
+          headers: {
+            Authorization: `Bearer ${requestToken?.value || ''}`,
+          },
+          cache: 'no-cache',
+        });
```

Comment:
noticed that there's another endpoint call that should be replaced with a server side function

Reply:
Updated on this commit [36a5218](https://github.com/inlivedev/inlive-room/pull/128/commits/36a52182807119a594c0a94cf22615d04a22cba9)